pkg/storage/mysql/mysqlstore_test.go

```diff
@@ -0,0 +1,587 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package mysql
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
```

Comment:
I am concerned about this usage.

Using this as a backend for the tests then places responsibility squarely on the maintainers to know how MySQL works. This carries the real risk of improper tests. Eg.: the current implementation will fail with keys of sufficient length.

Please - as a TODO - refactor to use a real MySQL instance.